Gemfile

```diff
@@ -10,6 +10,7 @@ gem 'rails', '~> 6.1.5.1'
 
 gem 'bootsnap', require: false
 
+gem 'rack', '~> 2.2.3.1'
```

Comment:
you don't need this

Reply:
This update was due to CI security

Reply:
is a dependency of rails, if you just do `bundle update rack` will update the gemfile.lock and you don't need to add the gem here

Reply:
Oh, good to know!

It is already fixed.